app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -524,18 +528,18 @@ def attributes_for_log
 
     attributes["ethnic_group"] = ethnic_group_from_ethnic
     attributes["ethnic"] = field_30
-    attributes["national"] = field_31
-    attributes["income1nk"] = field_32.present? ? 0 : 1
+    attributes["national"] = buyer_not_interviewed? ? 13 : field_31
+    attributes["income1nk"] = buyer_not_interviewed? || field_32.blank? ? 1 : 0
     attributes["income1"] = field_32
```

Comment:
Do we perhaps want to set `income1 ` to be nil if the buyer hasn't been interviewed?

Reply:
it's not mentioned in the ticket - I don't think we can guarantee that's correct info so we should leave as is